src/components/MessageView.tsx

```diff
@@ -42,6 +46,41 @@ export default function MessageView (props: { message: Message; id: string }) {
   return React.createElement(component, { ...message, id: props.id })
 }
 
+const handleDeleteMessage = (e, data) => {
+  const doDelete = confirm(`Are you sure you would like to delete the message '${data.message}'?`)
+  if (doDelete) {
+    deleteMessage(data.messageId)
+  }
+}
+
+const DeleteMessageView = (props: { messageId: string, message: string, id: string }) => {
```

Comment:
Unless I'm missing something, you don't seem to actually use `id` here, yeah? I think we can delete it?